tasks/main.yml

```diff
@@ -23,6 +23,12 @@
 - include: packages.yml
 - include: configuration.yml
 
+- name: Start libvirtd
```

Comment:
I think this could be better implemented as an handler triggered by "Install Nova Compute" for RedHat distros.